tests/test_cli.py

```diff
@@ -262,3 +263,33 @@ def test_lint_black_pyproj(pyproject, command, output, tmp_empty):
     result = runner.invoke(cli.cli, command)
 
     assert output in result.output
+
+
+@pytest.mark.parametrize(
+    "command, version, doc",
+    [
+        [
+            ["setup"],
+            None,
+            False,
+        ],
+        [
+            ["setup", "--version", "3.9"],
+            "3.9",
+            False,
+        ],
```

Comment:
can you test with other python version values?

Reply:
[Fixed here](https://github.com/ploomber/pkgmt/pull/74/files#diff-4e8715c7a425ee52e74b7df4d34efd32e8c92f3e60bd51bc2e1ad5943b82032eR268-R280) - Added more version values to test